backend/src/database/repositories/tenantRepository.ts

```diff
@@ -9,12 +9,28 @@ import Error400 from '../../errors/Error400'
 import { isUserInTenant } from '../utils/userTenantUtils'
 import { IRepositoryOptions } from './IRepositoryOptions'
 import getCleanString from '../../utils/getCleanString'
+import { createRedisClient, RedisClient } from '../../utils/redis'
+import { RedisCache } from '../../utils/redis/redisCache'
 
 const { Op } = Sequelize
 
 const forbiddenTenantUrls = ['www']
 
 class TenantRepository {
+  static _tenantCache: RedisCache
+
+  static _redisClient: RedisClient
```

Comment:
You shouldn't store `_redisClient` in the static variable since you only need it once.